.zuul.yaml

```diff
@@ -28,7 +28,6 @@
 - job:
     name: node-image-build
     abstract: true
-    pre-run: playbooks/pre.yml
```

Comment:
This looks wrong to me, why do you want to drop running this?

Reply:
You are right! I missed that and added  it now again. Although `create-image-flavor.sh` essentially automatically ensures that all the necessary preconditions are met, I still wanted to run it so that at least “ensure-docker” and “ensure-virtualenv” are active.

Is there an automatic resolution of dependencies in Zuul so that `pre.yml` is only triggered once when a series of images is created?

Reply:
No, each job is running on a fresh node, so you need to run pre.yml for each job